number-45/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+    <script>
+      let mooQuack = () => {
+        let number = 0;
+        for (let i = 5; i < 151; i++) {
+          if (i % 4 === 0 && i % 7 != 0){
+            console.log("Moo");
+            number++;
+          } else if (i % 7 === 0 && i % 4 != 0){
+            console.log("Quack");
+            number++;
+          } else if (i % 4 === 0 && i % 7 === 0){
+            console.log("MooQuack");
+          } else {
+            console.log(i);
+            number++;
```

Comment:
Looks good, minor error. Looks like you're not summing the right thing here. The total count of `moo`, `quack` and `mooquack`. Also consider creating `i % 4` and `i % 7` into a variable so you don't have to recalculate it each time.